src/Renderer/ConsoleRenderer.php

```diff
@@ -85,7 +83,7 @@ public function setFilterChain(FilterChain $filters)
     public function getFilterChain()
     {
         if (null === $this->__filterChain) {
-            $this->setFilterChain(new FilterChain());
+            $this->__filterChain = new FilterChain();
```

Comment:
Not much of a problem, but it may be possible that overrides to `setFilterChain()` make this change a potential issue.

I'll let this one roll though, as it's really an edge case.